rector.php

```diff
@@ -11,27 +11,25 @@
         __DIR__ . '/tests',
     ])
     ->withRootFiles()
-    ->withPHPStanConfigs([
-        __DIR__ . '/phpstan.dist.neon',
-        // rector does not load phpstan extension automatically so require them manually here:
-        __DIR__ . '/vendor/phpstan/phpstan-doctrine/extension.neon',
-        __DIR__ . '/vendor/phpstan/phpstan-symfony/extension.neon',
-    ])
-    ->withImportNames(importShortClasses: false)
-    ->withPreparedSets(codeQuality: true, doctrineCodeQuality: true)
+
     ->withPhpSets()
+    ->withPreparedSets(codeQuality: true, doctrineCodeQuality: true, symfonyCodeQuality: true, deadCode: true, codingStyle: true, instanceOf: true, typeDeclarations: true)
+    ->withImportNames(importShortClasses: false)
+    ->withAttributesSets(all: true)
 
     // symfony rules
     ->withSymfonyContainerPhp(__DIR__ . '/var/cache/website/dev/App_KernelDevDebugContainer.xml')
     ->withSets([
-        SymfonySetList::SYMFONY_CODE_QUALITY,
         SymfonySetList::SYMFONY_CONSTRUCTOR_INJECTION,
         // activate when doing updates:
         // SymfonyLevelSetList::UP_TO_SYMFONY_63,
-        // activate when doing updates:
-        // PHPUnitLevelSetList::UP_TO_PHPUNIT_90,
-        // PHPUnitSetList::PHPUNIT_91,
```

Comment:
I've bumped the PHPUnit to 10, which is the highest version with BC changes so far, so no need for extra upgrade sets.

Reply:
What is strange is that a basic symfony still installs PHPUnit 9:

```bash
composer create-project symfony/skeleton
cd skeleton
composer require test
```

Need todo some research why it is so 🤔

Reply:
I don't think flex + related hacks are maintained much.  Composer require works well:

```bash
composer create-project symfony/skeleton
cd skeleton
composer require phpunit/phpunit
```

![image](https://github.com/user-attachments/assets/fc47acd6-d6e8-4b3b-8904-9b0fd02008c9)